src/settings.h

```diff
@@ -28,7 +28,7 @@
 #define LCD_RST     6
 #define LCD_DC      7
 #define LCD_CS      5
-#define TOUCH_CS    14
+//#define TOUCH_CS    14
```

Comment:
I think you have a merge conflict here, so you'll need to merge dev-v2 into this branch